portfolio/src/main/webapp/script.js

```diff
@@ -20,11 +20,20 @@ const typewriterLetterDelayMs = 100;
 const typewriterWordDelayMs = 1000;
 const typewriterLoadDelayMs = 1000;
 
+<<<<<<< HEAD
+=======
+const logInPrompt =
+    'To simplify form entry, and to verify your identity, please log in.';
+const logOutPrompt = 'All done? Log out whenever you\'d like';
+
+// Start the typewriter effect when page loads
+>>>>>>> Hide form when logged out, show when logged in
```

Comment:
Oops, looks like you have a merge conflict that was left unresolved.

Reply:
Have no clue how I didn't see this - thanks for pointing this out